src/v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -77,6 +77,8 @@ const processTrackEvent = async (metadata, message, destination) => {
   const payload = constructPayload(message, updatedMapping);
 
   payload.partialFailure = true;
+  // ref: https://developers.google.com/google-ads/api/docs/release-notes#:~:text=Added%20job_id%20as,UploadConversionAdjustmentsRequest
+  payload.jobId = metadata.jobId;
```

Comment:
Can't we support properties.jobId as well? Some customer might try to provide jobId from their own. 
CC @shrouti1507 @krishna2020

Reply:
can you tell what's the use-case of customer for providing jobId ?

Reply:
Customer can maintain some unique numbers on their database and send that as jobId.